backend/climateconnect_main/settings.py

```diff
@@ -149,3 +149,7 @@
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
     'PAGE_SIZE': 100
 }
+
+SITE_ROOT = os.path.dirname(os.path.realpath(__file__))
+MEDIA_ROOT = SITE_ROOT+'/../media/'
+MEDIA_URL = '/media/'
```

Comment:
Hmmmm.... I rethink after our my last review. I think MEDIAL_URL should be pointed at GCP Storage. All media should be stored separate from server in cloud storage.